NutritionTracker/Models/Ingredient.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
+
+namespace NutritionTracker.Models
+{
+    public class Ingredient
+    {
+
+        [Key]
+        public int Id { get; set; }
+        [Required]
+        [StringLength(50, MinimumLength = 3, ErrorMessage = "Indgedient name cannot be longer than 50 character or shorter than 3 character.")]
+        public string ingredientName { get; set; }
+
+        [Required]
+        [Range(0, 20000)]
+        public int calories { get; set; }
```

Comment:
Calories could potentially be a calculated value based on the macros the user puts in for the ingredient and wouldn't need to be stored in the database.

``` csharp
public readonly int calories => this.CalculateCalories();

private int CalculateCalories()
{
    return (this.protein + this.carbs) * 4 + this.fats * 9;
}
```